src/libostree/libostree-devel.sym

```diff
@@ -20,7 +20,7 @@
 /* Add new symbols here.  Release commits should copy this section into -released.sym. */
 LIBOSTREE_2020.2 {
 global:
-  someostree_symbol_deleteme;
+  ostree_repo_commit_modifier_sepolicy_from;
```

Comment:
Bikeshed: how about `ostree_repo_commit_modifier_set_sepolicy_from` to match the existing `ostree_repo_commit_modifier_set_sepolicy`?

Hmm, and maybe also we just spell it out entirely it's from what, i.e. `from_commit` ?

Reply:
Pretty long identifier but...eh, sure.